.editorconfig

```diff
@@ -13,3 +13,5 @@ trim_trailing_whitespace = false
 [{.travis.yml,npm-shrinkwrap.json,package.json}]
 indent_style = space
 indent_size = 2
+[Makefile]
+indent_style = tab
```

Comment:
Yes, I was again misleading myself :D 

Thought you edit another file. It's okay :D